gemsfx/src/main/java/com/dlsc/gemsfx/CalendarView.java

```diff
@@ -411,6 +411,30 @@ public final void setDisableYearDropdownButton(boolean disableYearDropdownButton
         this.disableYearDropdownButton.set(disableYearDropdownButton);
     }
 
+    /**
```

Comment:
Please move the description to the property accessor method "dateFilterProperty". It is a convention by JavaDocs to look for JavaFX properties there. Also add a comment related to the fact that date ranges can still include such disabled dates but that they can not be used to start or end a range definition.